src/style.css

```
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: normal;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+form {
+  position: relative;
+  margin: 20px 0;
+}
+
+input {
+  box-sizing: border-box;
+  font-family: inherit;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  color: #3d4e61;
+  width: 100%;
+  border-radius: 4px;
+}
+
+.search__input:hover {
+  background: #fff;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 20%);
+}
+
+.search__big__img {
+  position: absolute;
+  height: 19px;
+  width: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.search__big__input {
+  padding-left: 62px;
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.search__small {
+  position: relative;
+}
+
+.search__small__img {
+  position: absolute;
+  height: 11px;
+  width: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.search__small__input {
+  height: 42px;
+  padding-left: 33px;
+  background: #fff;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: 900;
+}
```

```suggestion
}

```
add an empty line of the end of file